src/confd/yang/infix-if-bridge@2024-03-06.yang

```diff
@@ -297,20 +314,43 @@ submodule infix-if-bridge {
           leaf-list untagged {
             type if:interface-ref;
             description "The set of ports in the untagged set for VLAN.";
-          }
+            must "current() = ../../../../if:name
+                  or (/if:interfaces/if:interface[if:name = current()]/bridge-port/bridge = ../../../../if:name
+                      and not(../tagged[contains(., current())]))" {
+              error-message "Port is not a member of this bridge or already set as tagged member in the same VLAN.";
```

Comment:
Splitting this up into two separate `must` expressions would maybe give better feedback to the user.

Reply:
Yeah I tried that before, but then veth inference in the CLI stopped working.  So let's try that again after the big rebase on Buildroot 24.02 + sysrepo/libyang upgrade party.

Reply:
Ahh right! My amnesia is getting worse, sorry about that! :smile: